sale_order_line_price_history/wizards/sale_order_line_price_history.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="sale_order_line_price_history_view_form" model="ir.ui.view">
+        <field name="name">sale.order.line.price.history.view.form</field>
+        <field name="model">sale.order.line.price.history</field>
+        <field name="arch" type="xml">
+            <form>
+                <group col="4">
+                    <field name="partner_id" />
+                    <field name="product_id" readonly="1" />
+                    <field name="include_commercial_partner" />
+                    <field name="include_quotations" />
+                </group>
+                <group string="Sale history">
+                    <field name="line_ids" nolabel="1" readonly="1">
+                        <form>
+                            <div
+                                class="oe_button_box"
+                                name="button_box"
+                                attrs="{'invisible': [('history_sale_order_line_id', '=', False)]}"
+                            >
+                                <button
```

Comment:
Put this button as well in the tree view

Reply:
When the popup opens, the wizard record hasn't been created yet, so the button on those lines will be disabled (normal odoo behavior).

Reply:
I'm not saying that. I just want to have the button "Set price" on the tree view of the wizard with an icon, only when the related sale line already exists.

Reply:
I wrote the limitation in the roadmap in the migration commit, please, check it out.